.env

```diff
@@ -15,9 +15,17 @@ TRAEFIK_DASHBOARD_PORT_OUT=8080
 HOSTNAME="localhost"
 BROKER_URI="test.mosquitto.org"
 
-TRAEFIK_DASHBOARD_USER=
-TRAEFIK_DASHBOARD_PASS=
+# OAUTH & GITHUB FORWARD AUTH
+OAUTH_SECRET=
+GITHUB_AUTH_URL=https://github.com/login/oauth/authorize
+GITHUB_TOKEN_URL=https://github.com/login/oauth/access_token
+GITHUB_USER_URL=https://api.github.com/user
+GITHUB_CLIENT_ID=
+GITHUB_CLIENT_SECRET=
+WHITELISTED_EMAILS=
 
+# SERVICE PORTS
+FORWARD_AUTH_PORT=4181
```

Comment:
This port is not open afaik

Reply:
I assume we don't need to expose this port. It is just a port number forward-auth is running in the container.